kubernetes-pipeline/templates/jenkins/jenkins-deployment.yaml

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: extensions/v1beta1
```

Comment:
@Aaquiff let's improve this Kubernetes Deployment spec to the latest API version in the next release. Please create an appropriate issue for this.

Reply:
Changed to apps/v1